lib/active_storage_support/base64_attach.rb

```diff
@@ -0,0 +1,17 @@
+# Helper method to decode a base64 file and create a StringIO file
+module ActiveStorageSupport
+  module Base64Attach
+    module_function
+
+    def attachment_from_base64(data:, filename: Time.current.to_i.to_s)
+      return if data.to_s.empty?
```

Comment:
why do you need this line having the one below?

Reply:
This is what Carrierwave base64 does. But yeah, it makes sense not to have it since `data.is_a?(String)` would return false if it data was null. Good catch